params.yml

```diff
@@ -179,3 +182,20 @@ args:
 
   ## Tools versions
   polycli_version: v0.1.42
+
+  ## blutgang configuration.
+  blutgang_image: makemake1337/blutgang:latest
+
+  blutgang_http_address: 0.0.0.0:55555
+  blutgang_admin_address: 0.0.0.0:55556
```

Comment:
I think you can get rid of those parameters and use `blutgang_rpc_port` and `blutgang_admin_port`.

Reply:
I also think that would be cleaner for sure.. but I'm referring to this variable in the `config.toml` file - is there a way to do string concat (e.g. `0.0.0.0:` + `blutgang_rpc_port` here?) 

blutgang seems to work with `0.0.0.0:<port>` and doesn't with other addresses like `127.0.0.1:<port>`

Reply:
Yeah it would work! Here is an example: https://github.com/0xPolygon/kurtosis-cdk/pull/87/files#r1579152214